packages/paypal-commerce-integration/src/paypal-commerce-alternative-methods/create-paypal-commerce-ratepay-payment-strategy.ts

```diff
@@ -5,13 +5,15 @@ import {
 
 import createPayPalCommerceIntegrationService from '../create-paypal-commerce-integration-service';
 import PaypalCommerceRatepayPaymentStrategy from './paypal-commerce-ratepay-payment-strategy';
+// import { LoadingIndicator } from '@bigcommerce/checkout-sdk/ui';
```

Comment:
Don't forget to remove commented code from the file

Reply:
removed